src/main/java/micdoodle8/mods/galacticraft/core/Constants.java

```diff
@@ -2,14 +2,14 @@
 
 public class Constants
 {
-    public static final String MOD_ID_CORE = "GalacticraftCore";
+    public static final String MOD_ID_CORE = "GRADLETOKEN_MODID";
     public static final String MOD_ID_PLANETS = "GalacticraftMars";
     public static final String MOD_NAME_SIMPLE = "Galacticraft";
     
     public static final String MOD_ID_GALAXYSPACE = "GalaxySpace";
     public static final String MOD_ID_GREGTECH = "gregtech";
```

Comment:
isn't it available in gregtech dependency constants?

Reply:
same for galacticraft

Reply:
Maybe but why would you want to use that?

Reply:
I don't think making the modid controlable via the buildscript is a good idea though, we are not going to change it and it would be more error-prone.

Reply:
You CANNOT! It is used before gregtech is loaded. To be evaluated in @Mod for example.

Reply:
Just to not give a chance of having a typo/forgetting to change string when name somehow updated. But since it's unavailable, then yeah, nvm

Reply:
I repeat, the modid should *under no circumstances* be changed! **The Gradletoken is not needed at all.**